library/hawkular_alerts_group_trigger.py

```diff
@@ -155,49 +192,70 @@ def required_updates(self, trigger, name, severity, enabled):
             updates["enabled"] = enabled
         return updates
 
-    def update_group_trigger(self, trigger, updates):
+    def set_group_trigger_conditions(self, group_id, conditions):
+        """ Set the conditions for the group trigger
+        """
+        gc = hawkular.alerts.GroupConditionsInfo()
+        for c in conditions["conditions"]:
+            name = c.pop("name")
+            condition = hawkular.alerts.Condition(c)
+            condition.trigger_mode = conditions["trigger_mode"]
+            condition.context = {'name': name}
+            gc.addCondition(condition)
+        try:
+            self.client.create_group_conditions(group_id, conditions["trigger_mode"], gc)
+        except Exception as e:
+            self.module.fail_json(msg="Failed to set group trigger conditions. Error: {error}".format(error=e))
+        self.changed = True
+
+    def update_group_trigger(self, trigger, updates, conditions):
```

Comment:
If so, it'll have a boolean flag for conditions.
I'm thinking about it cause then I won't be able to check `if not updates:` and move to the `update_group_trigger` method (cause it might only need an update in the conditions)